### PR TITLE
Changed requirements to pull edx-i18n-tools from PyPI and .gitignore …

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
…to include en files

@clintonb @bderusha Please review

### CODE REVIEW

**Comment by clintonb** [comment]
👍 Remember to commit the English files.

File: requirements/local.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@ django-debug-toolbar==1.3.2
 
 # i18n
 transifex-client==0.10
-git+https://github.com/edx/i18n-tools.git@v0.1.4#egg=i18n_tools==0.1.4
+edx-i18n-tools>=0.3.3
```
Comment (nedbat) [inline requirements/local.txt]: Don't we pin versions?

**Comment by iivic** [comment]
@clintonb @mjfrey Please review or confirm merge.

**Comment by clintonb** [comment]
Include the English .po files.

**Comment by iivic** [comment]
Review once more.

File: programs/conf/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,1293 @@
...
+#: programs/apps/programs/models.py:273
+msgid ""
+"Duplicate course run modes are not allowed for course codes in a program."
+msgstr ""
+
+#: programs/apps/programs/models.py:279
+msgid "Invalid course key."
+msgstr ""
+
+#: venv/lib/python2.7/site-packages/alabaster/layout.html:55
```
Comment (clintonb) [inline programs/conf/locale/en/LC_MESSAGES/django.po]: Something is wrong here. Why are all of these strings from your virtualenv included?

Comment (iivic) [inline programs/conf/locale/en/LC_MESSAGES/django.po]: I don't know what went wrong. I noticed this in other PR-s as well. I will just try again and add the missing .mo files.

Comment (iivic) [inline programs/conf/locale/en/LC_MESSAGES/django.po]: I created the virtualenv in the same folder where the MakeFile is. I moved the virtualenv outside the project now it looks good.

**Comment by clintonb** [comment]
👍

**Pull request merged**